CMakeLists.txt

```diff
@@ -510,6 +520,16 @@ if(OPENSSL_AARCH64_P256)
   add_definitions(-DOPENSSL_AARCH64_P256)
 endif()
 
+# For Android build, CMAKE_SYSTEM_PROCESSOR is sometimes not defined in CMAKE_TOOLCHAIN_FILE.
+if(ANDROID)
```

Comment:
Do we want to say NOT CMAKE_SYSTEM_PROCESSOR here in addition to ANDROID? Or just always set it?

Reply:
PR in `dockcross` has been merged: https://github.com/dockcross/dockcross/pull/480
Remove the definition of `CMAKE_SYSTEM_PROCESSOR` here.